_posts/2020-04-10-notes-from-community-call-2-april-2020.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Notes from community call - 2 April 2020"
```

Comment:
This should be 'Standard community call'

Reply:
The title is harmonized with previous community calls. If this change is made, should I also change previous blog posts?

Reply:
I think for now this is fine, I didn't realise we haven't merged the 'splitting the community calls' yet.

Reply:
Same here